File: programming/2.2/main.py
from math import atan, tan, exp, sqrt, log, atan
from random import randint


y = 0
x = randint(0, 10)
x1 = 0.011
x3 = 1.546
K = 0.800
L = 7.145

a = (K * x**5 - 10 * exp(-x))**(1/3)
b = (log(L))**1/3 * tan(x)

condition = a**2 + b**2

if condition > 0.1:
  y = atan(a - 5 * b)

elif condition <= 0.1:
  y = atan(5 * a + 3 * b)

print('Result:', y)

File: programming/2.2/main.cpp
#include <iostream>
#include <cmath>


using namespace std;

int randint(int min, int max) {
  srand(time(NULL));
  return rand() % (max + 1 - min) + min;
}

int main() {
  int K = 2, x = randint(10, 20);
  float L = 7.39 * pow(10, 3);
  float x1, x3, a, b, y;

  a = (L * x) / (pow(x, 5) * (exp(K) - 1));
  b = sqrt(exp((x-1) / sin(x))) + cos(x);

  float condition = pow(a, 2) + pow(b, 2);
  if (condition > 0.1) {
    y = atan(a - 5 * b);
  }
  else if (condition <= 0.1) {
    y = atan(5 * a - 3 + b);
  }

  cout << "Result: " << y;

  
  return 0;
}